Dockerfile

```diff
@@ -29,7 +29,22 @@ WORKDIR /usr/local/src
 COPY requirements.txt .
 RUN pip install -r requirements.txt
 
+RUN apt update
+RUN apt -y install tesseract-ocr
+RUN pip install pytesseract
```

Comment:
Why not add `pytesseract` and the other required packages to `requirements.txt` instead?